client/src/featureform/serving.py

```diff
@@ -129,7 +129,7 @@ def _create_channel(self, host, insecure, cert_path):
     def training_set(self, name, variation, include_label_timestamp, model: Union[str, Model] = None):
         return Dataset(self._stub).from_stub(name, variation, model)
 
-    def features(self, features, entities, model: Union[str, Model] = None):
+    def features(self, features, entities, model: Union[str, Model] = None, params: list = []):
```

Comment:
think @aolfat had a post about using empty lists as defaults and how you shouldnt do it

Reply:
https://featureform-workspace.slack.com/archives/C022U1JHDB3/p1679592789364619

Reply:
Also how does this handle multiple on-demand features in one call? are params a list of tuples?

Reply:
params will be used by the user. It can essentially be a list of whatever they want.